database_schema.md

```diff
@@ -136,6 +136,76 @@ seat_id: {
 }
 ```
 
+### Minutes Item
+```
+minutes_item: {
+    minutes_item_file_id: str
+    minutes_item_id: str
+    name: str
+    uri: str
+    title: str
+    matter: {
+      matter_id: str
+      matter_name: str
+      matter_title: str
+    }
```

Comment:
If I understand correctly, this block can be entirely optional? As there will exist minutes items that are not matters.

Reply:
Yeah it could be completely optional